lib/ramble/ramble/application.py

```diff
@@ -325,6 +329,18 @@ def set_template(self, is_template):
         """Set if this instance is a template or not"""
         self.is_template = is_template
 
+    def set_repeats(self, repeats):
+        """Set if this instance will use repeats
+
+        repeats = (is_repeat_base, n_repeats)
+
+        is_repeat_base (Bool): True if this is the base experiment of a repeat set,
```

Comment:
This doesn't look like it's an argument anymore.

Reply:
Can you explain what you mean?

Reply:
Oh sorry, I misunderstood t his comment. You mean the input argument is a tuple, and the tuple has is_repate_base / n_repeats as the two values.

I thought this was describing the arguments to the method.

Can you update the doc string here to explain that a bit more (could just be with plain English) for the next reader?

Reply:
That's right, input is a tuple. I've reworded to make it more clear.

Reply:
nit; Maybe (to make this more consistent with other argument docs):

"""
repeats (tuple): consists of:
is_repeat_base( Bool): ...
num_repeats (Int): ...
"""